cli/schema_describe.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/sourcenetwork/defradb/client"
+)
+
+func MakeSchemaDescribeCommand() *cobra.Command {
+	var name string
+	var root string
+	var versionID string
+
+	var cmd = &cobra.Command{
+		Use:   "describe",
+		Short: "View schema description.",
+		Long: `Introspect schema types.
+
+Example: view all schema
```

Comment:
typo: "schemas"

Reply:
Not a typo, but, after googling, I am incorrect anyway and this needs changing (possibly in a few places) - I actually thought `schema` was both plural and singular, like `sheep` and `sheep` :sweat_smile: 

- [ ] Schema is not plural